requirements.txt

```diff
@@ -10,7 +10,8 @@ pytest-benchmark
 requests
 tabulate
 git+https://github.com/huggingface/pytorch-image-models.git@730b907
-git+https://github.com/huggingface/transformers.git@6c26faa
+# this version of transformers is required as per this page https://huggingface.co/mistralai/Mistral-7B-Instruct-v0.1
+transformers==4.34.1
```

Comment:
This will be shared across all transformer models, and we sort of want to keep a consistent version as in the pytorch docker build: https://github.com/pytorch/pytorch/blob/main/.ci/docker/ci_commit_pins/huggingface.txt. Is there a reason we want to update the transformer revision?

Reply:
This is the latest published version and (unfortunately) this model requires the latest one. The .dev versions didn't work for me. Is there a better place to add this? I thought of adding to requirements.txt for this model but then when the test for this model runs, this version of transformer will get installed in the environment and all future model tests will end up using this version, no?

Reply:
All the model tests will share the same version of transformer, regardless before or after this model. Our policy is that we can accept if it is later than `6c26faa`. But we cannot accept it if `4.34.1` is earlier than `6c26faa`. Can you help double check?

Reply:
Commit [6c26faa](https://github.com/huggingface/transformers/commit/6c26faa159b79a42d7fa46cb66e2d21523351987) was done on September 8th. Release 4.34.1 was [released](https://github.com/huggingface/transformers/releases/tag/v4.34.1) a few days ago. It's a patch release.

Reply:
Sounds good, thanks for checking this! Still we need to fix the CI error, then the PR looks good to me.